### PR TITLE
chore: update & document release process

### PR DESCRIPTION
**Pull request by mydiemho**
swimlanes-9edfa565b100bdc65903c210b69a2eaa

GitHubRelease

### CODE REVIEW

File: azure-pipelines/create-release.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: "GitHub & Web Release [$(SourceBranchName)] - $(Date:yyyyMMdd)$(Rev:.r)"
```
Comment (wbreza) [inline azure-pipelines/create-release.yml:1]: Was the old process not working?  Looks like a full rewrite on the pipelines?

File: azure-pipelines/templates/create-web-release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+      - task: AzureFileCopy@3
+        displayName: 'AzureBlob File Copy to $DEV_STORAGE_ACCOUNT'
+        inputs:
+          SourcePath: './build'
+          azureSubscription: $(azureSubscription)
+          destination: azureBlob
+          storage: $(DEV_STORAGE_ACCOUNT)
+          containerName: '$web'
+        # condition: eq(variables['Build.SourceBranch'], 'refs/heads/develop')
```
Comment (wbreza) [inline azure-pipelines/templates/create-web-release.yml:36]: Should this line be commented out?

File: azure-pipelines/create-release.yml (hunk 1 above)
Comment (mydiemho) [reply azure-pipelines/create-release.yml:1]: a lot of the old process was manual.  A few of AzDO task has been updated to support the functionality we need so I removed the manual pieces.

File: azure-pipelines/templates/create-web-release.yml (hunk 2 above)
Comment (mydiemho) [reply azure-pipelines/templates/create-web-release.yml:36]: good catch

**Pull request merged**